stress/common/src/main/java/alluxio/stress/common/SummaryStatistics.java

```diff
@@ -39,6 +41,12 @@
    */
   public SummaryStatistics() {
     // Default constructor required for json deserialization
+    mTimePercentileMs = new float[101];
+    Arrays.fill(mTimePercentileMs, 0);
+    mTime99PercentileMs = new float[StressConstants.TIME_99_COUNT];
+    Arrays.fill(mTime99PercentileMs, 0);
+    mMaxTimeMs = new float[StressConstants.MAX_TIME_COUNT];
+    Arrays.fill(mMaxTimeMs, 0);
```

Comment:
Can you describe why these are necessary? When are these values missing during json serialization?

Reply:
Sometimes, when there is no data, we just serialize a default constructed instance, and these fields were all `null`.